### DISCUSSION TITLE
如何使用webpack的umd externals的方式减小包大小，cdn链接使用哪个引入可以达到要求

### DISCUSSION THREAD

**Discussion by hcxmj** [ant-design/ant-design#44024] [Q&A]
### What problem does this feature solve?

目前使用包大小分析工具发现占1M左右，希望可以使用webpack的umd externals的方式减小包大小，提高下载速度

### What does the proposed API look like?

希望文档上可以给出如何使用cdn配合webpack减小包大小

**Comment by Wxh16144** [comment]
cdn 不是更大么，antd5 现在全是 js 文件，按理说应该有  tree-shaking 的吧
